### ISSUE TITLE
Rework sync screen

### ISSUE THREAD

**Issue by mark-prins** [msupply-foundation/open-msupply#2522]
## What went wrong? 😲
 
The sync screen is a little hard to follow, and when I've watched people use it, and heard feedback, they always click both buttons when they need to sync manually.

![Screenshot 2023-11-21 at 2 51 47 PM](https://github.com/msupply-foundation/open-msupply/assets/9192912/c10e8465-6bb4-41e6-ad80-7f94592a4a91)

This shows to me that the screen is not clearly explaining what the intention is. 
If users are always syncing both the site and the user status, then we should just combine the two in the manual sync process. If there's a good reason to have them separated, and I think there is, then keep them separate but make it clearer.

## Expected behaviour 🤔
Some suggestions:

- add explanatory text below the heading to say what type of data is synchronised and what clicking the button will do
- make the user synchronisation section separate in some way
  - add tabs, or
  - an accordion, or
  - change colours of the user sync, and/or
  - add an icon for the user
- change the text on the button and/or the icon so that the user sync is distinct

**Comment by mark-prins** [comment]
Perhaps not really a 'bug'. Just a thing.

The decision is to make it a single button, and to add the user sync as part of the timeline indication. Put the user first; if all you want to do is sync the user, you might not want to wait for the full sync to happen.

**Comment by nisha-dangol** [comment]
## Testing
Tested with and without a suitable central server running in v1.6.0.rc2 in server exe.

I'm PASSING this one as I see the old (single sync button) again it's syncing the user permissions as well with that same Sync button.
<img width="1484" alt="image" src="https://github.com/msupply-foundation/open-msupply/assets/59193490/7e586035-ce0d-4861-89eb-feccf8694f09">
